demos-europe/demosplan-core
# feat (DPLAN-3268): code inspection fixes

### Ticket
DPLAN-3268

Some suggestions from code inspections.

### How to review/test
Code Review. Test the export in the interface. Should work like before.

Delete the checkbox if it doesn't apply/isn't necessary.
